knip.ts

```diff
@@ -5,7 +5,8 @@ import type { KnipConfig } from 'knip'
 const sveltePreprocessor = sveltePreprocess()
 
 const config: KnipConfig = {
-	ignore: ['**/*.d.ts'],
+	ignore: ['**/*.d.ts', '**/schemas.ts', 'src/lib/objects/hello-world/index.ts'],
```

Comment:
Why are the `'**/schemas.ts', 'src/lib/objects/hello-world/index.ts'` ignored?

Reply:
Because knip reports unused exports when only a type is imported, which is the case with the schema files and the `HelloWorldStore` in case of the hello world `index.ts`.

Reply:
I see. This is a good one to report to the `knip` dev. I'll do that